tests/testsuite/package.rs

```diff
@@ -7025,3 +7025,89 @@ src/main.rs
 "#]])
         .run();
 }
+
+#[cargo_test]
+fn git_core_symlinks_false() {
```

Comment:
Can you do before/after commits for this test?

Reply:
Thanks for the catch! Done.